src/test/kotlin/net/atos/zac/mail/MailServiceTest.kt

```diff
@@ -65,7 +53,7 @@ class MailServiceTest : BehaviorSpec({
         loggedInUserInstance,
     )
 
-    beforeEach {
+    afterEach {
```

Comment:
Sure about this? `afterEach` I think means that it will not be checked for the first test that is run?

Reply:
Worked the same way in this test, but you're right. Will bring this back to `beforeEach`